code/game/machinery/poolcontroller.dm

```diff
@@ -100,7 +100,7 @@
 		if(drownee.losebreath > 20)	//You've probably got bigger problems than drowning at this point, so we won't add to it until you get that under control.
 			return
 
-		add_attack_logs(src, drownee, "Drowned", isLivingSSD(drownee))
+		add_attack_logs(src, drownee, "Drowned", isLivingSSD(drownee) ? null : ATKLOG_ALL)
```

Comment:
Remove this entire isLivingSSD condition please, the automatic category sorter should handle this well.

Reply:
Ahh, right, it spams you. This is fine then.